product_docs/docs/pgd/5/rel_notes/pgd_5.4.0_rel_notes.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: "EDB Postgres Distributed 5.4.0 release notes"
+navTitle: "Version 5.4.0"
+---
+
+Released: xx xxx 2024
+
+EDB Postgres Distributed version 5.4.0 is a minor version of EDB Postgres Distributed.
+
+!!! Important Recommended upgrade
+We recommend that all users of PGD 5 upgrade to PGD 5.4. See [PGD/TPA upgrades](../upgrades/tpa_overview) for details.
+!!!
+
+## Highlights of EDB Postgres Distributed 5.4.0
+
+* Group Commit and Parallel Apply are now compatible with each other.
+
+## Compatibility
+
+!!! Note EDB Server version compatibility
+This version requires the recently released Postgres versions 14.10, 15.4,
+or 16.1 (or later) of EDB Postgres Advanced Server or EDB Postgres Extended
+Server. No such restrictions exist for PostgreSQL Server.
+
+Package managers on Debian, RHEL, and SLES pull in the required EDB Postgres Advanced Server or EDB Postgres Extended 
+upgrades with an upgrade of EDB Postgres Distributed.
+!!!
+
+## Features
+
+| Component | Version | Description  | Addresses |
+|-----------|---------|-------------|-----------|
+| PGD | 5.4.0 | Group Commit and Parallel Apply can work together with a dedicated writer for Group Commit. | | /* TBC (To Be Committed) */
+
+## Enhancements
+
+| Component | Version  |Description | Addresses |
+|-----------|---------|-------------|-----------|
+| BDR       | 5.4.0   | Reduced WAL file retention when decoding worker enabled by using more compact LCR files. | | /* 00-wal-reduction-with-LCR */
+| BDR       | 5.4.0   | Reinforce features that avoid data loss when reconciling failed transactions || /* TBC */
+
+## Bug fixes
+
+| Component | Version |Description  | Addresses |
+|-----------|---------|-------------|-----------|
+| BDR       | 5.4.0   | Fixed memory leaks when running a query on some or all nodes. |  | /* 10-Memory-Leak-Run-On-All-Nodes */
+| BDR       | 5.4.0   | Resolved an issue of high CPU usage for concensus processes | RT97649 | /* 20-High-CPU-usage-consensus */
+| BDR       | 5.4.0   | Improved WAL retention logic when a part_node occurs. |  | /* 30-group-slot-and-wal-retention */
+| BDR       | 5.4.0   | Witness nodes will automatically not synchronize structure when joining a group |  | /* 40-BDR-4507-bdr_join_node_group-witness-no-synchronize-structure */
+| BDR       | 5.4.0   | bdr.drop_node now returns the OID of the dropped node, not `1`. |  | /* 40-BDR-4565-bdr_drop_node-fix-return-value */
+| BDR       | 5.4.0   | bdr.drop_node now returns the OID of the dropped node, not `1`. |  | /* 40-BDR-4565-bdr_drop_node-fix-return-value */
+| BDR       | 5.4.0   | bdr.create_node() / bdr.alter_node() now give a hint when an invalid node kind is used. | | /* 41-BDR-4652-bdr_create_node-HINT */
+| BDR       | 5.4.0   | Fixed transactions PREPARE/COMMIT/ABORT order with Parallel Apply enabled. || /* 67-Fix-Prepared-Parallel-Apply */
```

Comment:
Missing 70-BDR-4397-ddl-gucs and 90-Increase-Default-raft-keep-min-entries and 92-BDR-4493-fix-group-snapshot-restore

Reply:
92 is already there
| BDR       | 5.4.0   | Restoring a node group from a consensus snapshot correctly applies option changes (number of writers, streaming and apply_delay) to local subscriptions. || /* 92-BDR-4493-fix-group-snapshot-restore */